src/components/Accordion/AccordionItem/AccordionItem.jsx

```diff
@@ -15,21 +15,41 @@ export const AccordionItem = ({id, label, icon, onClickToOpen, onClick, onClickT
         }
 
         onClick(e);
```

Comment:
If we want to keep callback here, one is enough - onClickToOpen and onClickToClose are superfluous ( but pass open as a parameter to onClick )